jobs/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = patterns('',
     url(r'^$', views.jobs, name='jobs'),
+    url(r'^details/(?P<id>\d+)$', views.job_details, name='job_details'),
```

Comment:
Suggest we leave out the "details" part of the url - so we have /jobs (for the listing of all jobs) and /jobs/1 (for the detail view)